src/frida/repl.py

```diff
@@ -443,6 +502,42 @@ def _create_repl_script(self):
     }
 };
 """
+        def _update_truststore(self, record):
+            trust_store = self._get_or_create_truststore()
+            trust_store.update(record)
+
+            config_dir = os.path.join(os.path.expanduser('~'), '.frida')
+            codeshare_trust_store = os.path.join(config_dir, "codeshare-truststore.json")
+
+            with open(codeshare_trust_store, 'w') as f:
+                f.write(json.dumps(trust_store))
+
+        def _get_or_create_truststore(self):
+            '''
+            Look for a trust store located at ~/.frida/codeshare-truststore.json, which holds known fingerprints of scripts previously
+            run before, prompting if the fingerprint changes (due to a code change pushed to codeshare).
+            :return:
+            '''
```

Comment:
I also notice you don't have a lot of comments, I'm happy to remove this if you'd like as it's not exactly rocket science to figure out what this does.